src/nomad_material_processing/vapor_deposition.py

```diff
@@ -219,12 +220,6 @@ class Substrate(ArchiveSection):
             "index": 1,
         },
     )
-    substrate_specimen = Quantity(  ############ my humble proposal
-        description="""
-        The thin film stack that is being evaporated on.
-        """,
-        type=ThinFilmStack,
-    )
```

Comment:
Proposal sounds okay but why are you deleting it?

Reply:
I don't see it being repeated here?

Reply:
we can normalize then in components all this entries

Reply:
I see, sure. But this was never repeating and shouldn't be in my opinion.

Reply:
Wait, why are there components in the PLDStep?

Reply:
For sure! my mistake, only layer is repeated.
So Iwill chenge this part as discussed. Nice.

Reply:
We are not in PLDstep now, am I mistaking something?

Reply:
Yes we are, in a subsection of PLDStep that is.

Reply:
This is in the Substrate class that we compose inside VaporDepositionStep. so it ends up also in PLDstep
Was it different before?

Reply:
I think we are misunderstanding each other. I'll give you a call after I made my coffee and explain.